src/components/Settings/AccountTab.react.js

```diff
@@ -22,6 +22,7 @@ import styled from 'styled-components';
 import { setUserSettings, uploadAvatar } from '../../apis';
 import defaultAvatar from '../../images/defaultAvatar.png';
 import CircularProgress from '@material-ui/core/CircularProgress';
+import { userNameRegEx } from '../../utils/regex';
```

Comment:
Can you use this constant wherever applicable to remove redundacy? Like for eg - Signup modal.
Same for passwordRegex.